### PR TITLE
Callautomation/juntuchen/release/ga1

### PR DESCRIPTION
**Pull request by juntuchen-msft**
**List of changes for GA SDK:** 

Autorest.md to point to main stable swagger on api-specs. Wait for this PR to be finished: [callautomation/2023-03-06-ga1 by minwoolee-msft · Pull Request #23091 · Azure/azure-rest-api-specs (github.com)](https://github.com/Azure/azure-rest-api-specs/pull/23091) 

Autorest.md’s tag to be pointing to stable version tag. tag: package-2023-03-06 

Update ServiceVersion in ClientOption: 2023-03-06 

Regenerate Autorest with above GA swagger 

README.md to be updated to accommodate GA version.  

Remove –prerelease tag 

Remove any section that presents any feature that should not be included 

Changelog.md: Update Release history to be latest version (1.0.0), Please look at each languages guideline’s section “Package version number” for details on how each language update this. 

Changelog.md: With version updated and new section, add all breaking changes. 

[GA_Release_Checklist.docx](https://microsoft.sharepoint-df.com/:w:/t/ServerCalling/EV3lTIkFsQdJpy7s3J0gutkB0fBxdz-p9Ljtgdur9l8aEQ?e=NNTSK8) (look at this page for what needs to be included in the changelog) 

(Nice to have) Need more samples under Samples forlder. 

**Feature to be excluded:** 

Remove Custom HMAC support 

MediaStreaming features (note this is also included in part of CreateCall/AnswerCall) 

Play,PlaytoAll,Recognize,Cancel are only media feature required – remove all other media methods 

Only Recognize DTMF is to be included in GA. Exclude other Recognize modes such as Choices, Speech, and continous 

Remove all new events that were added with new media features 

ChoiceResult 

ContinousDTMFRecognition 

SendDTMF 

All CustomContext related in Signaling methods 

All AzureCognitive related features 

All mute/unmute features 

All play source except FileSource (SSMLSource, TextSource to be excluded) 

Recording’s External Storage related features (such as BlobStorage) 

Remove all test associated with removed features 

By Removing method and models above, also remove all orphaned model/enums that are no longer in use (such as, GenderType), Use APIView to view any orphaned models

### CODE REVIEW

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/AnswerCallOptions.java
Comment (minwoolee-msft) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/AnswerCallOptions.java:1]: Missing answeredBy

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CallConnectionProperties.java [hunk 1]
```diff
@@ -152,23 +149,14 @@ public String getCallConnectionId() {
...
-     */
-    public String getMediaSubscriptionId() {
-        return mediaSubscriptionId;
-    }
-    
     /**
      *  Get identity that answered the call
      * @return identity that answered the call
      */
     public CommunicationUserIdentifier getAnsweredByIdentifier() {
```
Comment (minwoolee-msft) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CallConnectionProperties.java:156]: remove Identifier at the end

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CreateCallOptions.java [hunk 2]
```diff
@@ -20,21 +20,11 @@ public final class CreateCallOptions {
...
     private String operationContext;
 
-    /**
-     * Media Streaming Configuration.
-     */
-    private MediaStreamingOptions mediaStreamingOptions;
-
     /**
      * Constructor
      * @param callInvite Call invitee information.
```
Comment (minwoolee-msft) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CreateCallOptions.java:30]: CallInvite Model still has customercontext voip headers

File: sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CreateGroupCallOptions.java
Comment (minwoolee-msft) [inline sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CreateGroupCallOptions.java:1]: still has voip/sipheaders

**Comment by minwoolee-msft** [comment]
media's ResonCode file still has few reasoncode related to speech, such as: 

>  public static final ReasonCode SPEECH_OPTION_MATCHED = fromReasonCode(8545);
public static final ReasonCode SPEECH_OPTION_NOT_MATCHED = fromReasonCode(8547);

**Review by ikumarapeli-msft — APPROVED** [review]

**Review by minwoolee-msft — APPROVED** [review]

**Pull request merged**